libdnf/CMakeLists.txt

```diff
@@ -19,8 +19,8 @@ add_definitions(-DLIBSOLV_SOLVABLE_PREPEND_DEP)
 include_directories(.)
 
 # build libdnf.so
-add_library(libdnf SHARED ${LIBDNF_SOURCES})
-set(DNF_SO_VERSION 3)
+add_library(libdnf SHARED ${LIBDNF5_SOURCES})
+set(DNF_SO_VERSION 1)
 set_target_properties(libdnf PROPERTIES OUTPUT_NAME "dnf")
```

Comment:
Also the OUTPUT_NAME should be changed to dnf5. That will result in `libdnf5.so.1` so file name.

Reply:
thanks, done